### DISCUSSION TITLE
Azure Private DNS

### DISCUSSION THREAD

**Discussion by dixonsj** [polymind-inc/acmebot#706] [Q&A] [answered]
Hi, just wondering if there are any examples of utilizing the Azure Private DNS provider. How does the challenge work if it is a private DNS zone?

**Comment by shibayan** [comment, marked as answer]
I have never used Private DNS myself, but I added it because it is in demand when operating Private PKI as in #518, and it is easy to implement in terms of SDK.

Public PKI does not work because it cannot resolve challenges.

**Reply by dixonsj** [reply]
I thought as much. Thanks for the awesome work btw.
